modules/gce_docker_registry_cache/cloud-init.bash

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# vim:filetype=sh
+set -o errexit
+set -o pipefail
+
+main() {
+  : "${ETCDIR:=/etc}"
+  : "${VARTMP:=/var/tmp}"
+
+  export DEBIAN_FRONTEND=noninteractive
+
+  eval "$(tfw printenv docker)"
+  __setup_docker
+  __install_registry
+}
+
+__setup_docker() {
```

Comment:
I think that this is now redundant/outdated given what `tfw` provides by default. See the squignix setup for a more recent example https://github.com/travis-ci/terraform-config/blob/master/modules/gce_squignix/cloud-init.bash

Reply:
Where is the docker config being passed to the docker daemon?